ang/civicase/ActivitiesCalendar--decorator.js

```diff
@@ -5,6 +5,17 @@
     $provide.decorator('uibDaypickerDirective', function ($controller, $delegate) {
       var datepicker = $delegate[0];
 
+      /**
+       * After the link function is executed, emits an event that signals
+       * that the directive is compiled and attached to the DOM
+       */
+      datepicker.compile = function () {
+        return function ($rootScope) {
+          datepicker.link.apply(this, arguments);
+          $rootScope.$emit('uibDaypicker::compiled');
```

Comment:
Since it is under civicase extension, we should rename the event `civicase::uibDaypicker::compiled` since its the standard we are following.